### ISSUE TITLE
BUG: Enchanted works locally but not through ngrok/mobile.

### ISSUE THREAD

**Issue by joewinke** [AugustDev/enchanted#78]
I am using ngrok static domain and had everything working. Then today the app doesn't work on iphone - I have tested on two iphones in different geographies. Then I ran Enchanted mac app on the computer running ngrok/ollama and it works fine. Same server url, I don't think I've changed anything but can't figure out why it won't work

Ollama running on 11434:
![Screenshot 2024-04-24 at 12 08 25 PM](https://github.com/AugustDev/enchanted/assets/797297/1a467590-5084-4ed0-a253-05d575a02f7f)

Ngrok running with addr = 11434:
![Screenshot 2024-04-24 at 12 08 33 PM](https://github.com/AugustDev/enchanted/assets/797297/190dfc64-2f34-4e58-9d46-495a759ae0c4)

What could cause this // how to debug? Thank you!

**Comment by joewinke** [comment]
OK I dived deeper in and found out that its because I've used up all the bandwidth on my ngrok plan:
![Screenshot 2024-04-24 at 12 15 08 PM](https://github.com/AugustDev/enchanted/assets/797297/2ac02bb3-9fe8-4a97-8086-0e72c4b970d9)

I only actually have sent probably sent less than 50 prompts from enchanted mobile through the ngrok tunnel as I mostly use it locally on my machine, which the above evidences skips ngrok completely, so the Enchanted app is constantly sending https requests. Is this intentional? Could this be throttled down to not consume all this bandwidth? Or is this number of requests normal?
